### PR TITLE
feat(ui): add background layout to the kitchenSink

### PR DESCRIPTION
**Pull request by adimachmali**

### CODE REVIEW

File: packages/common/global-types/kaltura-player/player-config.d.ts [hunk 1]
```diff
@@ -10,195 +10,205 @@
...
+    export interface ContribFonts {
+      fontFamily: string;
+      testingFont?: TestingFontOptions;
+      downloadData?: {
+        name: string;
+        url: string;
+      };
+    }
+
+    export interface KitchenSinkConfig {
```
Comment (esakal) [inline packages/common/global-types/kaltura-player/player-config.d.ts]: 1. in `contrib-plugin-configs.ts` adjust `get playerConfig(): KalturaPlayerTypes.PlayerConfig | ContribConfig`
2. in the same file add
```

interface ContribConfig {
  contrib?: {
    ui?: {
      ....
    }
  } 
} 
```
and move relevant parts that both you and Shai wrote into this interface

@ainvoner fyi

Comment (adimachmali) [reply packages/common/global-types/kaltura-player/player-config.d.ts]: done

File: packages/ui/src/components/banner-container/banner-container.tsx [hunk 2]
```diff
@@ -1,9 +1,11 @@
 import {Component, h} from 'preact';
 import {ContribLogger, getContribLogger} from '@playkit-js-contrib/common';
 import * as styles from './_banner-container.scss';
+import BannerConfig = KalturaPlayerContribTypes.BannerConfig;
```
Comment (esakal) [inline packages/ui/src/components/banner-container/banner-container.tsx]: i don't think this import is needed, can you check?

File: packages/ui/src/components/banner-container/banner-container.tsx [hunk 3]
```diff
@@ -1,9 +1,11 @@
...
 
 export interface BannerContainerProps {
   onClose: () => void;
+  bannerConfig: BannerConfig;
```
Comment (esakal) [inline packages/ui/src/components/banner-container/banner-container.tsx]: consider asking specific the `theme` if possible, to make the public API of the component better

File: packages/ui/src/components/kitchen-sink/kitchen-sink.tsx [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import {Component, h} from 'preact';
 import {ContribLogger, getContribLogger} from '@playkit-js-contrib/common';
 import * as styles from './_kitchen-sink.scss';
-import KitchenSinkConfig = KalturaPlayerTypes.PlayerConfig.KitchenSinkConfig;
+import KitchenSinkConfig = KalturaPlayerContribTypes.KitchenSinkConfig;
```
Comment (esakal) [inline packages/ui/src/components/kitchen-sink/kitchen-sink.tsx]: is this needed? try to remove

File: packages/ui/src/font-manager.ts [hunk 5]
```diff
@@ -4,12 +4,12 @@ import {
   ObjectUtils,
   PlayerContribRegistry,
 } from '@playkit-js-contrib/common';
-import ContribFonts = KalturaPlayerTypes.PlayerConfig.ContribFonts;
-import TestingFontOptions = KalturaPlayerTypes.PlayerConfig.TestingFontOptions;
 import PlayerConfig = KalturaPlayerTypes.PlayerConfig;
+import ContribFonts = KalturaPlayerContribTypes.ContribFonts;
```
Comment (esakal) [inline packages/ui/src/font-manager.ts]: is this needed? try to remove

File: packages/ui/src/kitchen-sink-item.tsx [hunk 6]
```diff
@@ -12,7 +12,7 @@ import {
   KitchenSinkEvents,
 } from './kitchen-sink-manager';
 import {KitchenSink} from './components/kitchen-sink';
-import KitchenSinkConfig = KalturaPlayerTypes.PlayerConfig.KitchenSinkConfig;
+import KitchenSinkConfig = KalturaPlayerContribTypes.KitchenSinkConfig;
```
Comment (esakal) [inline packages/ui/src/kitchen-sink-item.tsx]: is this needed? try to remove

File: packages/ui/src/kitchen-sink-manager.tsx [hunk 7]
```diff
@@ -16,7 +17,8 @@ import {PresetNames} from './preset-item-data';
 import {KitchenSinkContainer} from './components/kitchen-sink-container/kitchen-sink-container';
 import {KitchenSinkAdapter} from './components/kitchen-sink-adapter';
 import {ManagedComponent} from './components/managed-component';
-import KitchenSinkConfig = KalturaPlayerTypes.PlayerConfig.KitchenSinkConfig;
+import {UpperBarItem} from './upper-bar-item';
+import KitchenSinkConfig = KalturaPlayerContribTypes.KitchenSinkConfig;
```
Comment (esakal) [inline packages/ui/src/kitchen-sink-manager.tsx]: is this needed? try to remove

File: packages/ui/src/components/banner-container/banner-container.tsx (hunk 2 above)
Comment (adimachmali) [reply packages/ui/src/components/banner-container/banner-container.tsx]: done

File: packages/ui/src/components/banner-container/banner-container.tsx (hunk 3 above)
Comment (adimachmali) [reply packages/ui/src/components/banner-container/banner-container.tsx]: done

**Pull request merged**